utils/docker/Dockerfile.checkpatch

```diff
@@ -5,26 +5,31 @@
 #
 
 # Pull base image
-FROM fedora:33
+FROM fedora:latest
 
-ARG CACHEBUST
+# Intermittent cache-bust.  Used to reduce load on the actual CACHEBUST later.
+ARG CB0
 
 # Install OS updates and package.  Include basic tools and daos dependencies
 # that come from the core repo.
 # Clean up any repos afterwards to save space.
-RUN yum -y upgrade && 						\
-    yum -y install codespell diffutils file findutils git	\
+RUN dnf -y upgrade && 						\
+    dnf -y install codespell diffutils file findutils git	\
 	python3-clustershell python3-pygithub python3-numpy	\
 	python3-paramiko python3-pylint python3-pyxattr		\
 	python3-tabulate ShellCheck &&				\
     dnf -y module enable avocado:latest &&			\
-    dnf -y module install avocado &&				\
-    yum clean all
+    dnf -y module install avocado
 
 ARG UID=1000
 
 # Add DAOS users
 RUN useradd --uid $UID --user-group --create-home --home /home/daos daos
 
 WORKDIR /home/daos
+
+# Actual CACHEBUST.  Do this every time so the system is up to date.
+ARG CACHEBUST
+RUN dnf update && dnf -y upgrade && dnf clean all
```

Comment:
https://dnf.readthedocs.io/en/latest/command_ref.html#upgrade-command-label

`update` is a deprecated alias for `upgrade`.

Reply:
Done.